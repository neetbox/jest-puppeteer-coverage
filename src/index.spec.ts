import { renderHTML, setTitle } from ".";

test("renderHTML", async () => {
  renderHTML(page);
  const content = await page.content();
  expect(content).toMatchSnapshot();
});

test("setTitle", async () => {
  setTitle(page, "hello");
  const content = await page.content();
  expect(content).toMatchSnapshot();
});
