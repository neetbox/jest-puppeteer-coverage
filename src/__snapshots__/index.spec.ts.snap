// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index renderHTML 1`] = `
"<!DOCTYPE html><html lang=\\"en\\"><head>
      <meta charset=\\"UTF-8\\">
      <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\">
      <meta http-equiv=\\"X-UA-Compatible\\" content=\\"ie=edge\\">
      <title>Document</title>
    </head>
    <body>
      <div id=\\"app\\">hello</div>
    
    
  </body></html>"
`;

exports[`index setTitle 1`] = `
"<!DOCTYPE html><html lang=\\"en\\"><head>
        <meta charset=\\"UTF-8\\">
        <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\">
        <meta http-equiv=\\"X-UA-Compatible\\" content=\\"ie=edge\\">
        <title>hello</title>
      </head>
      <body>
      
    </body></html>"
`;
